lib/app/sequence_base.rb

```diff
@@ -920,6 +920,49 @@ def find_slice_by_values(element, values)
         end
       end
 
+      def validate_bindings(bindings, resources)
```

Comment:
I know `sequence_base` has traditionally been a dumping ground for shared methods but we have to stop doing this.  We need to be moving in the direction of `sequence_base` purely being 'Inferno DSL' definitions (e.g. `test`, `omit`, `metadata`), otherwise we will never be able to separate the 'platform' from the 'tests'.

It looks like this method is only used in the bulk_data_group_validation sequence, so can we just move it there and not touch sequence_base?

Reply:
This method can be used by single patient tests. I would like to create a separate ticket to update test generator to use this method for terminology validation

Reply:
If you are going to do a bit of refactor to get the single patient query API tests sharing the same code in another ticket, lets just keep this method out of sequence base and then move it to that shared place as part of that work (almost definitely not into sequence_base but that other shared/utility/helper module, along with the dozen+ other us-core specific methods we have in there).  We need to move in the right direction from here on out.

Reply:
The function is moved to bulk_data_group_validation_sequence.